fltk/datasets/cifar10.py

```diff
@@ -3,6 +3,8 @@
 from torchvision import transforms
```

Comment:
This file should actually be removed to avoid confusion. Only distributed/cifar10.py is used

Reply:
Done, it's removed